examples/bn254/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "bn254-inner"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+bn254-core = { path = "../../core" }
+risc0-zkvm = { path = "../../../../risc0/zkvm", default-features = false, features = ["std", "unstable"] }
+substrate-bn = "0.6.0"
```

Comment:
if its a pre-release version, maybe better to lock it down to exactly 0.6.0 ` = "=0.6.0"`